### PR TITLE
chore(button): updated outline button color to look different than plain text

### PR DESCRIPTION
**Pull request by briedinger**
"Ghost buttons" `btn-outline-primary border-0` are often use to get an affordance of clicking, with hover effects and all, without having a full button style. This change updates the color for primary to remain blue, so it behaves like a clickable hyperlink and not plain text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Questions? Ask them in the [forum](https://groups.google.com/forum/#!forum/opens
 
 * Java 1.7.0+
 * Node/NPM
+* Python
```
Comment (bradh) [inline README.md]: Looks like you've got a lot of changes that are unrelated in this PR. Maybe a bad merge or rebase?

**Comment by briedinger** [comment]
Closing, cherry-picked this commit to a new branch to clean things up.
